igcollect/artfiles.py

```diff
@@ -31,21 +37,23 @@ def parse_args():
 
 def main():
     args = parse_args()
-    artfiles_url = (
-        'https://{}:{}@dcp.c.artfiles.de/api/stats/get_estats.html'
-        .format(args.http_user, args.http_pass)
-    )
-    data = urlopen(artfiles_url)
+    request = Request('https://dcp.c.artfiles.de/api/stats/get_estats.html')
+    credentials = '{}:{}'.format(args.http_user, args.http_pass)
+    base64credentials = base64.b64encode(credentials.encode())
+    request.add_header('Authorization',
+                       'Basic {}'.format(base64credentials.decode()))
+    data = urlopen(request)
     template = args.prefix + '.{}.{}.{}.{} {} ' + str(int(time()))
```

Comment:
Would prefer to have named template entries, though not in PR scope. So, feel free to do it, but not a must have.

Reply:
Fixed 😃